go.mod

```diff
@@ -39,6 +39,11 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/go-playground/locales v0.14.1 // indirect
+	github.com/go-playground/universal-translator v0.18.1 // indirect
```

Comment:
Seems like these should be one stanza down, with all our other indirect dependencies.